src/components/HeaderBlock.ts

```diff
@@ -9,15 +9,13 @@ export interface HeaderBlockProps
 export interface HeaderBlockSpec {
   type: 'header';
   text: ReturnType<typeof PlainText>;
-  emoji: true;
 }
 
 export const HeaderBlock: Block<HeaderBlockProps, HeaderBlockSpec> = ({
   children,
   blockId,
 }) => ({
   type: 'header',
-  emoji: true,
```

Comment:
did this used to be an option on the slack header block?

Reply:
might have been but not anymore